service/tasks/driver.py

```diff
@@ -907,6 +908,9 @@ def _deploy_init_to(driverCls, provider, identity, instance_id,
         playbooks = ansible_deploy_to(instance.ip, username, instance_id)
         _update_status_log(instance, "Ansible Finished for %s." % instance.ip)
         logger.debug("_deploy_init_to task finished at %s." % datetime.now())
+    except AnsibleDeployException as exc:
+        logger.exception(exc)
+        _deploy_init_to.retry(exc=exc)
```

Comment:
There are still problems with network partitioning where it will eventually resolve. that's why we still retry some. But it's lowered to 10 retries and a max of 32 minutes.